nucleus/hk2/hk2-config/src/main/java/org/jvnet/hk2/config/ConfigParser.java

```diff
@@ -74,16 +74,27 @@
  * @author Kohsuke Kawaguchi
  */
 public class ConfigParser {
+
+    private static final Logger LOGGER = Logger.getAnonymousLogger();
```

Comment:
I don't like using anonymous logger - it cannot be configured well in logging.properties. Better would be `Logger.getLogger(ConfigParser.class.getName());`